### DISCUSSION TITLE
Jaeger with Clickhouse

### DISCUSSION THREAD

**Discussion by AshishM-Protean** [jaegertracing/jaeger#5542] [General]
Can you please help me implement Jaeger with Clickhouse?

**Comment by selfing12** [comment]
the latest version that supports working with CH is 1.57, for this grpc-plugin CH is used
https://github.com/jaegertracing/jaeger-clickhouse/tree/main
here is the configuration
https://github.com/jaegertracing/jaeger-clickhouse/blob/main/config.yaml
we include the binary file in query and in collector

**Reply by AshishM-Protean** [reply]
can we detailed documentation how we can configure this using kubernetes
